instrumentation/opentelemetry-instrumentation-aws-lambda/tests/test_aws_lambda_instrumentation_manual.py

```diff
@@ -40,10 +40,18 @@
 )
 from opentelemetry.util._importlib_metadata import entry_points
 
+from .mocks.alb_conventional_headers_event import MOCK_LAMBDA_ALB_EVENT
+from .mocks.alb_mutl_value_headers_event import (
```

Comment:
You have a typo in the file name, s/mutl/multi/

Reply:
Sorted 😄